docs/gadget-devel/gadget-ebpf-api.md

```diff
@@ -345,9 +345,15 @@ A new field containing the flags set will be added to the event:
 
 #### Annotations
 
+Following the list of field annotations available to customize the behavior and
+appearance of the bitfield fields:
+
 - `ebpf.formatter.enum`: Name of the new field. If the annotation is not set and the source field name has a `_raw` suffix, the target name will be set to the source name without that suffix.
 - `ebpf.formatter.bitfield.separator`: Separator used. Defaults to `|`.
 
+Further information about the field annotations can be found in the
+[metadata](metadata.md#fields) documentation.
```

Comment:
Do you think it'd be redundant to add this intro and reference to other enriched types with annotations (`gadget_timestamp`, `gadget_signal`, `gadget_errno`, `gadget_syscall` and `gadget_kernel_stack`)?